MIGRATIONS.md

```diff
@@ -8,10 +8,10 @@ Migrations are simple scripts which deploy or modify contracts. The goal of migr
 To create a new migration, run:
 
 ```sh
-yarn hardhat gen:migration --network kovan my_migration
+yarn hardhat gen:migration --network kovan --market 01-USDC my_migration
```

Comment:
I know we discussed this already, but given that these may be part of URLs etc, I think we should drop the prefix and only add a suffix to differentiate if/when needed